tools/myst/host/package.c

```diff
@@ -666,6 +666,17 @@ int _exec_package(
         goto done;
     }
 
+    /* Check --host-uds option */
+    if (cli_getopt(&argc, argv, "--host-uds", NULL) == 0)
+    {
+        fprintf(
```

Comment:
I realize there are many of these places where we use ``fprintf(stderr, ...)`` but the correct form is to use ``_err()``, which prefixes "argv[0]". This is a defacto standard in Linux where commands must prefix all errors with the name of the command (because the command can call another command and without the prefix it would be unclear which command printed the errors). For example:

```
$ ls xxx
ls: cannot access 'xxx': No such file or directory
```